simple/entities/sample/observations.csv

```diff
@@ -0,0 +1,21 @@
+entity,variable,date,value
+country/AFG,var1,2023,0.19
+country/YEM,var1,2023,0.21
+country/AGO,var1,2023,0.29
+country/ZMB,var1,2023,0.31
+country/ZWE,var1,2023,0.37
+country/ALB,var1,2023,0.5
+country/DZA,var1,2023,0.52
+country/AND,var1,2023,0.76
+country/AFG,var2,2023,6.0
+country/YEM,var2,2023,56.0
+country/AGO,var2,2023,6.0
+country/ZMB,var2,2023,34.0
+country/ZWE,var2,2023,76.0
+country/ALB,var2,2023,34.0
+country/DZA,var2,2023,92.0
+country/AND,var2,2023,9.0
+country/ASM,var2,2023,34.0
+country/AIA,var2,2023,42.0
+country/WLF,var2,2023,75.0
+country/ESH,var2,2023,65.0
```

Comment:
why are there more rows than input?

Reply:
This is because the input CSVs are "wide" (i.e. one column per SV) where the output CSV is "long" (i.e. one row per SV value).

Reply:
I see, there are two var in the input